server/src/domain/asset/response-dto/memory-lane-response.dto.ts

```diff
@@ -0,0 +1,7 @@
+import { AssetResponseDto } from './asset-response.dto';
+
+export class MemoryLaneResponseDto {
+  title!: string;
+
+  assets!: AssetResponseDto[];
```

Comment:
What is Dto? Data Transfer Object? What's the significance?

Reply:
We use DTO for openapi to generate  the strong type for request and response